mmv1/templates/terraform/examples/storage_managed_folder_nested.tf.erb

```diff
@@ -0,0 +1,15 @@
+resource "google_storage_bucket" "bucket" {
+  name                        = "<%= ctx[:vars]['bucket_name'] %>"
+  location                    = "EU"
+  uniform_bucket_level_access = true
```

Comment:
Please add `force_destroy = true` to this bucket so that it will be deleted even if there are still objects in it.

Reply:
done